tasks/eslint.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('eslint', {
+    app: {
+      eslintrc: '.eslintrc',
+      src: ['Gruntfile.js', 'tasks/*.js', 'src/*.js', 'scripts/*.js', 'lib/*.js']
```

Comment:
With the exclusions, this would still have 4 elements: 

```
src: [
  '**/*.js',
  '!node_modules/**/*',
  '!examples/**/*,
  '!test/**/*'
]
```

Since we clubbed src and lib into 1 folder anyway, I think having 4 folders specified explicitly is better.